### PR TITLE
Modal-Rating comps in Docs

### PR DESCRIPTION
**Pull request by MiheerTamkhane**
Added modal and rating comp in docs site.
- modal
- rating

### CODE REVIEW

**Review by sumitch29 — APPROVED** [review]
looks good to me

**Review by manojkumar8400 — APPROVED** [review]
good to go.

File: components/modal/modal.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+.ct-modal-header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  flex-wrap: wrap;
+  padding: 1rem 0;
+  border-bottom: 1px solid var(--gray-300);
+}
+.modal-close {
```
Comment (horsemaker) [inline components/modal/modal.css]: This should be `ct-modal-close`?

**Review by horsemaker — APPROVED** [review]

File: components/modal/modal.css (hunk 1 above)
Comment (MiheerTamkhane) [reply components/modal/modal.css]: ahh let me fix that. Thanks buddy

**Pull request merged**